packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -956,7 +975,38 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     }
   }
 
-  void _handleDelete({ required bool forward }) {
+  /// Computes the text after a delete event with a alt or meta modifier
+  String _computeBeforeTextWithBlockBackspace({ 
```

Comment:
To make it easier to read the `_handleDelete` method, I extracted `_computeBeforeTextWithBlockBackspace
` and `_computeBeforeTextWithBackspace` to different methods